azure-pipelines/templates/steps/automation-cert.yml

```diff
@@ -16,22 +16,22 @@ steps:
     azureSubscription: 'AuthSdkResourceManager' # string. Workload identity service connection to use managed identity authentication
     KeyVaultName: 'msidlabs' # string. Required. The name of the Key Vault containing the secrets.
     #setting secrets filter to fetch only MSIDLABCertificate cert from the vault
-    SecretsFilter: 'LabVaultAccessCert' # string. Required. Specifies the secret to download. Use '*' for all secrets.
+    SecretsFilter: 'LabAuth' # string. Required. Specifies the secret to download. Use '*' for all secrets.
     #RunAsPreJob: false # boolean. Make secrets available to whole job. Default: false.
 - task: PowerShell@2
   displayName: Install Automation Cert
```

Comment:
There is a similar yml file in common repo, we will need these changes there too.
Or better if we can use this yml everywhere and delete the other one.

Reply:
Yeah, i'm trying to use the common one everywhere. Can you clarify what this line is doing?

Import-PfxCertificate -FilePath $pfxPath -CertStoreLocation Cert:\CurrentUser\My

It seems to cause issues with ubuntu images, if it's something we don't need, i can remove it rather than having to update agents all over, which might break other stuff.

Reply:
This tries to install the cert on the agent machine's certificate store. 
This is only applicable for JVM based tests, where tests can use the cert from the certificate store and don't need to pass the cert path.

Reply:
Also, my preference would be to use the one in android-complete everywhere.